src/SubscribeMe/Subscriber/MailjetSubscriber.php

```diff
@@ -68,10 +82,78 @@ public function subscribe(string $email, array $options, array $userConsents = [
                     return $body['Data'][0]['ContactID'];
                 }
             }
-        } catch (RequestException $exception) {
+        } catch (ClientExceptionInterface $exception) {
             throw new CannotSubscribeException($exception->getMessage(), $exception);
         }
 
         return false;
     }
+
+    /**
+     * @see https://dev.mailjet.com/email/guides/send-api-v31/#use-templating-language
+     * @param array<EmailAddress> $emails
+     * @param array $variables
+     * @param string $templateEmail
+     * @return string
+     */
+    public function sendTransactionalEmail(array $emails, array $variables, string $templateEmail): string
+    {
+        $recipients = array_map(function (EmailAddress $emailAddress) {
+            return [
+                'email' => $emailAddress->getEmail(),
+                'name' => $emailAddress->getName(),
+            ];
+        }, $emails);
+
+        if (empty($emails)) {
+            throw new \InvalidArgumentException('Emails information missing');
+        }
+
+        if (empty($variables)) {
+            throw new \InvalidArgumentException('Variables missing');
+        }
+
+        if (empty($templateEmail)) {
+            throw new \InvalidArgumentException('Template Id missing');
+        }
+
+        if (!is_string($this->getApiKey())) {
+            throw new \InvalidArgumentException('ApiKey is not a string');
+        }
+
+        if (!is_string($this->getApiSecret())) {
+            throw new \InvalidArgumentException('ApiSecret is not a string');
+        }
+
+        $body = [
+            'Messages' => [[
+                'To' => $recipients,
+                'Variables' => $variables,
+                'TemplateID' => $templateEmail,
+                'TemplateLanguage' => true,
+            ]]
+        ];
+
+        if (!is_string(json_encode($body))) {
+            throw new \InvalidArgumentException('Body missing');
+        }
+        $body = $this->getStreamFactory()->createStream(json_encode($body));
```

Comment:
You are encoding body twice !

```suggestion
        $body = $this->getStreamFactory()->createStream(json_encode($body, JSON_THROW_ON_ERROR));
```